src/app/loadout-builder/generated-sets/SetStats.m.scss

```diff
@@ -87,5 +87,5 @@
 }
 
 .boostedValue {
-  color: $blue;
+  color: var(--theme-accent-secondary);
```

Comment:
Since these values are specifically boosted by `Font of ...` mods, I think we should continue to use blue here. Might be worth introducing a Sass variable.